app/queues/notifications/expiration-notification.server.ts

```diff
@@ -0,0 +1,189 @@
+// Third party modules
+import { Worker, Queue } from 'bullmq';
+
+// Internal modules
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { addNotification } from './notifications.server';
+import { prisma } from '~/db.server';
+
+// Types
+import type { ExpirationStatusPayload } from './record-cert-expiration-data.interface';
+import type { NotificationData } from './notifications.server';
+
+enum Type {
+  Certificate = 'certificate',
+  Record = 'record',
+}
+
+// name for the queue
+const expirationNotificationQueueName = 'expiration-notification';
+
+const updateNotificationStatus = (type: string, id: number) => {
+  // throw if any of the arguments are missing
```

Comment:
The way your types are defined in 22, this can never happen.

Reply:
changed this to `RecordType`